R2s example add outputs (#1085)
pyne/pyne: docs/usersguide/r2s.rst

PyNE R2S example
****************

> Using a simple geometry as a example, here is how we perform (voxel) R2S calculation.

Reviewer: should we introduce the concept of voxel vs. sub-voxel before describing "voxel" R2s?

> The example geometry composed of four cubes of size 10x10x10 cm3. It comsists of two different material, the water (in blue) and the steel (in red). The upper left cube (in white) is void.

Reviewer: consists
```suggestion
The example geometry is composed of four cubes of size 10x10x10 cm3. It consists of two different materials, the water (in blue) and the steel (in red). The upper left cube (in white) is void.
```

> There is a isotropic single energy (14 MeV) neutron point source in the center of the bottom right cube.

Reviewer: ```suggestion
There is an isotropic monoenergetic (14 MeV) neutron point source in the center of the bottom right cube.
```

The model is irradiated by a neutron source with intensitiy of 1e10 n/s for 3.5 days.

> The following content demonstrates the process of caculate the shutdown dose rate of this model at the time of 1 hour after shutdown.

Reviewer: ```suggestion
The following content demonstrates the process of caculating the shutdown dose rate of this model at the time of 1 hour after shutdown.
```

Example files could be found in 
`r2s_examples <https://github.com/pyne/data/blob/master/r2s_examples.tar.gz>`_.

> .. figure:: r2s_example_geometry.pdf

Reviewer: I think this probably needs to be a `png` and not a `pdf` to render properly...

    :align: center

    **Figure 1:** *Geometry of R2S example (X-Y cross section).*

> Build the model using trelis and set material group and export to `geom_without_material.h5m`.

Reviewer: ```suggestion
Build the model using Trelis (TM) and set material group and export to `geom_without_material.h5m`.
```
We may also want to add a link to Trelis and/or the DAGMC documentation

****************

Using a simple geometry as a example, here is how we perform R2S calculation.

> The example geometry is composed of four cubes of size 10x10x10 cm3. It consists of two different materials, the water (in blue) and the steel (in red). The upper left cube (in white) is void.

Reviewer: ```suggestion
The example geometry is composed of four cubes of size 10x10x10 cm\ :sup:`3`\ . It consists of two different materials, the water (in blue) and the steel (in red). The upper left cube (in white) is void.
```

   $ r2s.py setup

Modify important parameters in the 'alara_params.txt' and 'config.ini' according to the problem.

> Examples could be seen 'alara_params_example.txt' and 'config_example.ini' in 'r2s_examples/r2s_run'.

Reviewer: ```suggestion
Examples can be seen in 'alara_params_example.txt' and 'config_example.ini' in 'r2s_examples/r2s_run'.
```

By setting the 'sub_voxel' to be 'True', the user can perform sub-voxel R2S without any other change.
Prepare alara nuclide library, copy preinstalled data library from

> `ALARA/data <`r2s_examples <https://github.com/svalinn/ALARA/tree/master/data.>`_.

Reviewer: this seems to have too many back ticks (`)?

Author: That's true. It has been updated now.
